tests/cli/test_list.py

```diff
@@ -0,0 +1,172 @@
+"""
+Tests for the list file functions.
+
+.. raw:: html
+    <!--
+    * Copyright (C) 2024 Scribe
+    *
+    * This program is free software: you can redistribute it and/or modify
+    * it under the terms of the GNU General Public License as published by
+    * the Free Software Foundation, either version 3 of the License, or
+    * (at your option) any later version.
+    *
+    * This program is distributed in the hope that it will be useful,
+    * but WITHOUT ANY WARRANTY; without even the implied warranty of
+    * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    * GNU General Public License for more details.
+    *
+    * You should have received a copy of the GNU General Public License
+    * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+    -->
+"""
+
+import unittest
+from unittest.mock import patch, call
+from scribe_data.cli.list import (
+    list_languages,
+    list_data_types,
+    list_all,
+    list_languages_for_data_type,
+    list_wrapper,
+)
+from scribe_data.cli.main import main
+
+class TestListFunctions(unittest.TestCase):
+    @patch("builtins.print")
+    def test_list_languages(self, mock_print):
+        list_languages()
+        expected_calls = [
+            call(),
+            call('Language     ISO  QID    '),
+            call('-----------------------'),
+            call('English      en   Q1860  '),
+            call('French       fr   Q150   '),
+            call('German       de   Q188   '),
+            call('Italian      it   Q652   '),
+            call('Portuguese   pt   Q5146  '),
+            call('Russian      ru   Q7737  '),
+            call('Spanish      es   Q1321  '),
+            call('Swedish      sv   Q9027  '),
+            call('-----------------------'),
+            call(),
+        ]
+        mock_print.assert_has_calls(expected_calls)
+
+    @patch("builtins.print")
+    def test_list_data_types_all_languages(self, mock_print):
+        list_data_types()
+        expected_calls = [
+            call(),
+            call('Available data types: All languages'),
+            call('-----------------------------------'),
+            call('emoji-keywords'),
```

Comment:
Hmm - just noting something down here that came to mind just now..

Concerning the discussion we had on providing package options without PyICU and consequently the emoji functionality - omitting `emoji-keywords` as an available data type in the command output will probably be a good idea. With that, testing that the appropriate options are getting outputted given whether with/without PyICU might be good too.

...but! That's for when we get there :wink: